### ISSUE TITLE
Can not view Camel Route Diagram or Threads on WebLogic

### ISSUE THREAD

**Issue by lee120** [hawtio/hawtio#2422]
I have a small sample application with 2 camel routes.
Deploy camel application and hawtio-war as two separate war's.

When deploying to Jetty, hawtio views routes, Route Diagram correctly, I can start/stop flows etc.

When deploying to Weblogic, I can list the routes and most things runs ok but when expanding a route I don't see it's components and trying to view Route Diagram results in an empty page.

Threads panel is also missing when running inside Weblogic. But I can display the page, get a list of treads and statistics by manually entering url: http://localhost:8080/hawtio/#/threads

I use default settings when deploying the war's in Weblogic. Tried earlier hawtio versions (back to 1.5.0). Have not found any logged errors.

Versions used:
hawtio-default-offline-1.5.6.war
Oracle weblogic 12.1.3.0.0
Camel: 2.20.1
Java HotSpot(TM) 64-Bit Server VM - 1.8 (25.131-b11) by Oracle Corporation
![image](https://user-images.githubusercontent.com/23653925/34002967-5219c68c-e0f4-11e7-984e-c9cf1605c3ba.png)

Any ideas why the features that works fine on Jetty fails on WebLogic?
/Lars

**Comment by davsclaus** [comment]
Can you see anything in the console log of your web browser, or in the hawtio log tab? Also can you try to connect to your weblogic JVM via jconsole and see if you can browse the JMX tree and the Camel mbeans there without any problems.

**Comment by lee120** [comment]
No problem indicated in browser console or hawtio log tab.
MBeans seems to be displayed correctly, I can see producers, consumers, services, etc.

I have deployed an "old" Camel application, to same Weblogic instance. And Hawtio can display the Route Diagrams for those routes.
I tried using same (old) Camel version in the new application but same problem in Hawtio.
Only other difference I have seen is that the old application uses Spring, while the new one is set up without Spring. But maybe that is not the problem since "no-spring" works on Jetty.
